Rakefile

```diff
@@ -1,5 +1,13 @@
 require 'rake'
-require 'yaml'
+require 'pathname'
```

Comment:
This excerpt is from pe-razor-server: https://github.com/puppetlabs/pe-razor-server/blob/2018.1.x/Rakefile#L3-L10